Cargo.toml

```diff
@@ -15,6 +15,7 @@ members = [
     "genesis",
     "genesis-programs",
     "gossip",
+    "hardware-wallet",
```

Comment:
Looks like this can be called `remote-wallet`. I don't see a need to assume its implemented with separate hardware.  It's any wallet connected as a peripheral device, be it USB or Bluetooth.